src/restconfig/pom.xml

```diff
@@ -52,6 +52,23 @@
       <artifactId>javax.servlet-api</artifactId>
       <scope>provided</scope>
     </dependency>
+
+    <!-- add for rest api doc -->
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger2</artifactId>
+      <version>2.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>io.springfox</groupId>
+      <artifactId>springfox-swagger-ui</artifactId>
+      <version>2.9.2</version>
```

Comment:
In any case... [last commit](https://github.com/springfox/springfox/commits/master) is October 2020, two years and a half ago... the project is dead.

Regardless, I don't see how it can benefit the project, something like SpringFox should be able to figure out which resources and HTTP verbs are used, but it's unlikely to figure out the structure of the objects being traded as inputs and outputs, since it's all based on XStream with a long list of custom converters in the mix.